src/Laravel/Cashier/WebhookController.php

```diff
@@ -71,6 +69,7 @@ protected function handleInvoicePaymentFailed(array $payload)
     /**
      * Determine if the invoice has too many failed attempts.
      *
+     * @deprecated Use Stripe webhook 'customer.subscription.deleted' instead
```

Comment:
This annotation should be separated with an extra line after it, and the description should end in a fullstop.